pkg/bindings/artifacts/remove.go

```diff
@@ -27,6 +27,10 @@ func Remove(ctx context.Context, nameOrID string, options *RemoveOptions) (*enti
 
 	response, err := conn.DoRequest(ctx, nil, http.MethodDelete, "/artifacts/remove", params, nil)
 	if err != nil {
+		if options.GetIgnore() {
+			return &entities.ArtifactRemoveReport{}, nil
+		}
```

Comment:
Since this is already handled on the server side this check should not be needed at all.
And in fact it is somewhat wrong even because this silences all errors instead of the only by the option promised not exist error.